runtime/os_interface/DebugVariables_base.inl

```diff
@@ -61,7 +61,7 @@ DECLARE_DEBUG_VARIABLE(bool, LogTaskCounts, false, "Enables logging taskCounts a
 DECLARE_DEBUG_VARIABLE(bool, LogAlignedAllocations, false, "Logs alignedMalloc and alignedFree allocations")
 DECLARE_DEBUG_VARIABLE(bool, LogMemoryObject, false, "Logs memory object ptrs, sizes and operations")
 DECLARE_DEBUG_VARIABLE(bool, ResidencyDebugEnable, 0, "enables debug messages and checks for Residency Model")
-DECLARE_DEBUG_VARIABLE(bool, EventsDebugEnable, 0, "enables debug messages for events, virtual events, blocked enqueues, events trees etc.")
+DECLARE_DEBUG_VARIABLE(bool, EventsDebugEnable, 1, "enables debug messages for events, virtual events, blocked enqueues, events trees etc.")
```

Comment:
please do not enable any debug flags by default. these should stay disabled and enabled on demand.

if you need release driver with debug flags reader enable then you can add -DRELEASE_WITH_REGKEYS=1 to cmake command line.